src/CMakeLists.txt

```diff
@@ -303,8 +305,6 @@ endforeach()
 # Threading library -- uses IMPORTED target Threads::Threads (since CMake 3.1)
 set(CMAKE_THREAD_PREFER_PTHREAD TRUE)
 set(THREADS_PREFER_PTHREAD_FLAG TRUE)
-find_package(Threads REQUIRED)
```

Comment:
@ferdnyc MacOS Catalina and GCC 8.4 did not like this at all. The only way I could get a passing build was to remove this line. However, it still appears that Linux and Windows build find with this missing. I still need to test those builds, and make sure they actually work, but the compilation passes. Any thoughts on this one?

Reply:
@jonoomph Only just saw this, sorry. The threading library configs were only ever there for OpenMP, since apparently not linking with pthreads [could lead to link-time failures due to missing symbols](https://gitlab.inria.fr/openmp/libkomp/commit/cf0276b9ab4dba5d77a131fcf4061e5e2172d8eb). But I just ran a build with the current `CMakeLists.txt` and `/usr/lib64/libpthread.so` is right there next to `/usr/lib/gcc/x86_64-redhat-linux/10/libgomp.so` in the final linker command line, so clearly `OpenMP::OpenMP_CXX` is taking care of its dependencies now, the way it's supposed to.

In fact I just tried again after ripping out the variable-settings just above, and everything _still_ built exactly the way it was supposed to, so we can probably dump those as well just to avoid confusion. Might as well let OpenMP handle everything for itself.